implementations/rust/ockam/ockam_api/src/enroll/oidc_service.rs

```diff
@@ -230,7 +234,7 @@ impl OidcService {
 
                     // Note that a code 303 does not properly redirect to the success page
                     let response = Response::empty(302).with_header(
-                        Header::from_str("Location: https://account.ockam.io/device/success")
+                        Header::from_str(&format!("Location: {}/device/success", Self::authenticator_endpoint()))
```

Comment:
Similarly, should this and the next urls be part of the `OidcProvider` trait?

Reply:
In that case, that's the landing page when the authentication was successful. So it should be the same regardless of the provider.